sds_data_manager/lambda_code/SDSCode/indexer.py

```diff
@@ -232,15 +173,3 @@ def lambda_handler(event, context):
 
         # Write processing status data to DynamoDB.
```

Comment:
I think this was removed, so a rebase/merge of upstream/dev will remove some additional stuff for you.